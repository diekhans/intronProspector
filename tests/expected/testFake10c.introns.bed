fake10	34666	35513	sj0_GT/AG	1	-	34666	35513	0,128,0
fake10	130577	131433	sj1_GT/AG	1	+	130577	131433	0,128,0
fake10	131550	132801	sj2_GT/AG	1	+	131550	132801	0,128,0
fake210	132879	133326	sj3	1	+	132879	133326	64,128,64
fake310	133449	136100	sj4	1	+	133449	136100	64,128,64
fake310	136650	139594	sj5	1	+	136650	139594	64,128,64
