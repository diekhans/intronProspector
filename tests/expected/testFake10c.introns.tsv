chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites	name
fake10	34666	35513	-	1	0	0	81	20	0	GT/AG	sj0
fake10	130577	131433	+	1	0	0	72	29	0	GT/AG	sj1
fake10	131550	132801	+	1	0	0	85	16	0	GT/AG	sj2
fake210	132879	133326	+	1	0	0	80	17	0		sj3
fake310	133449	136100	+	1	0	0	15	85	0		sj4
fake310	136650	139594	+	1	0	0	75	26	0		sj5
